addons/spreadsheet_account/static/src/plugins/accounting_plugin.js

```diff
@@ -159,4 +160,28 @@ export class AccountingPlugin extends OdooUIPlugin {
         }
         return result;
     }
+
+    /**
+     * Gets the balance of a partner for given account code prefixes and over a given period
+     * @param {number[]} partner_ids ids of the partners
+     * @param {string[]} codes prefixes of the accounts codes
+     * @param {DateRange} dateRange start date of the period to search
+     * @param {number} offset end date of the period to search
+     * @param {number} companyId specific company to target
+     * @param {boolean} includeUnposted whether or not select unposted entries
+     * @returns {number | undefined}
+     */
+    getPartnerBalance(partner_ids, codes, dateRange, offset, companyId, includeUnposted) {
+        dateRange.year += offset;
+
+        const result = this.serverData.batch.get(
+            "account.account",
+            "get_partner_balance",
+            camelToSnakeObject({ partner_ids, codes, dateRange, companyId, includeUnposted })
+        );
+        if (result === false) {
+            throw new Error(_t("The partners' balance could not be computed."));
```

Comment:
`throw new EvaluationError(...)`
otherwise the spreadsheet thinks it's a programming error and report it as such.
Try `=ODOO.PARTNER.BALANCE(1)` => error is logged in the console and the cell error message says that the error is unexpected